include/DataFile.h

```diff
@@ -58,6 +58,8 @@ class EXPORT DataFile : public QDomDocument
 
 	virtual ~DataFile();
 
+	bool validate( QString extension );
```

Comment:
the ````DataFile```` in it's current state does not store the filename, I am trying to be as non invasive as possible.